peps/pep-9999.rst

```diff
@@ -0,0 +1,193 @@
+its desire to delegate decisions about the C API
+to such a working group, anticipating its formal establishment.
+
+Specification
+=============
+
+We propose the creation of a new group, the C API Working Group.
+This group will be responsible for overseeing and coordinating the development and
+maintenance of the Python C API.
+It will do this by establishing the principles underpinning the work
+and publishing guidelines that the core developers can refer to.
+
+The "operations and process" section below describes
+how the working group operates and how it is governed.
+
+Mandate
+-------
+
+The working group's mandate is to ensure that the Python C API is:
+
+* **Useful**: It should provide access to all relevant functionality.
+* **Ergonomic**: It should be easy to use.
+* **Fair**: It should serve the purposes of a variety of stakeholders.
+* **Stable**: It should avoid requiring users to update their code more than necessary.
+* **Evolvable**: It should give Python implementers enough freedom to change implementation details.
+* **Maintainable**: It should be easy to maintain and extend.
+
+For several of these items, a more comprehensive listing of constraints
+can be found in the PEP (number TO DO) which is based on the 
+`inventory of C API problems <https://github.com/capi-working-group/problems>`__.
+
+The stability requirement is not meant to prevent sweeping C API changes ---
+one of the working group's initial tasks is to settle the question of
+whether we intend any landmark changes to the C API in the coming five years,
+and if so, to select or write a suitable proposal.
+However, the working group is required to carefully consider the migration
+efforts required of package maintainers and other stakeholders.
+
+Operations and process
+----------------------
+
+The working group has at least three members,
+comprised of prominent Python core developers.
+The members should consider the needs of the various stakeholders carefully.
+
+The Steering Council would appoint the initial working group.
+There is no term limit for working group members.
+Working group members may resign their position at any time, for any reason.
+There is an expectation that the membership will change over time.
+
+To determine replacements,
+nominations will be collected from the core developer community.
+Self-nominations are allowed.
+The existing working group will then decide the replacement member(s)
+from the nominees.
+The expectation is that this would be done by fiat,
+but the working group can choose a replacement by any means they see fit,
+including a vote.
+
+The working group remains accountable to the Steering Council.
+At any point, for any reason, the Steering Council could
+(publicly or privately) make a specific change
+or request a non-specific change to the composition of the working group.
+
+We acknowledge that this is not a particularly democratic structure
```

Comment:
FWIW, I think this is a good structure for this group.

It may allay some concerns if we explicitly say before here that one of the responsibilities of the WG is to collect feedback and hear from other interested parties? I've seen people read a proposal of this shape before and assume that the appointees will only ever listen to themselves, and so they propose regular voting to avoid it. But I think if we explicitly say somewhere that part of the WG's job is to listen, it should soothe some of those concerns.

Reply:
I took this verbatim from Jelle's PEP. The typing folks did a lot of wordsmithing.

There are several mentions of stakeholders in the doc already, informing them, listening to them, fairly serving a variety of them. So I think that's covered.